src/AutoRest.CSharp/Common/AutoRest/Plugins/MemberRemoverRewriter.cs

```diff
@@ -93,6 +95,8 @@ public MemberRemoverRewriter(Project project, SemanticModel semanticModel, Immut
             {
                 if (attributeData.AttributeClass?.Equals(_suppressAttribute) == true)
                 {
+                    ValidateArguments(namedTypeSymbol, attributeData);
```

Comment:
Original idea was just to print a warning, but it remains invisible in a general log and generated code is still invalid. Hence I decided to terminate the codegen.